dss_datamover/master_application.py

```diff
@@ -62,6 +62,7 @@ class Master(object):
     def __init__(self, operation, config):
         manager = Manager()
         self.config = config
+        self.config_nfs = config["fs_config"]["nfs"]
```

Comment:
Wrong way of getting nfs shares when the variable self.fs_config is already initialized at the bottom.

Reply:
updated usage in latest commit.